File: expressions/financial_calculator.c
//Lila Shearer, Financial Calculator - C
#include <stdio.h>



int main(void){
    float income;
    float rent;
    float utilities;
    float groceries;
    float transportation;
    printf("Welcome to Financial Calculator!\nThis program is designed to calculate what percentages of your income that expenses, and savings take up, and show how much you have left to spend.\n");
    printf("What is your monthly income: ");
    scanf("%f",&income);
    printf("What is your monthly expense for rent or mortgage: ");
    scanf("%f",&rent);
    printf("What is your monthly expense for utilities: ");
    scanf("%f",&utilities);
    printf("What is your monthly expense for groceries: ");
    scanf("%f",&groceries);
    printf("What is your monthly expense for transportation: ");
    scanf("%f",&transportation);
    printf(" \n");
    float spending = (income-rent-utilities-groceries-transportation);
    float spending_percentage = (spending/income*100);
    float savings = (income/10);
    float savings_percentage = (savings/income*100);
    float rent_percentage = (rent/income*100);
    float utilities_percentage = (utilities/income*100);
    float groceries_percentage = (groceries/income*100);
    float transportation_percentage = (transportation/income*100);
    printf("Here are you results!\n");
    printf(" \n");
    printf("The amount of money you spend on your rent or mortage is $%.2f, which is %.1f%% of your monthly income.\n", rent, rent_percentage);
    printf("The amount of money you spend on utilites is $%.2f, which is %.1f%% of your monthly income.\n", utilities, utilities_percentage);
    printf("The amount of money you spend on groceries is $%.2f, which is %.1f%% of your monthly income.\n", groceries, groceries_percentage);
    printf("The amount of money you spend on transportation is $%.2f, which is %.1f%% of your monthly income.\n", transportation, transportation_percentage);
    printf("The amount of money you save is $%.2f, which is %.1f%% of your monthly income.\n", savings, savings_percentage);
    printf("The amount of money you have left to spend is $%.2f, which is %.1f%% of your monthly income.\n", spending, spending_percentage);


    return 0;
}

File: expressions/financial_calculator.py
# Lila Shearer, Financial Calculator - Python

print(" ")
# print statement that welcomes user and tells what program does
print("Welcome to Financial Calculator!\nThis program is designed to calculate what percentages of your income that expenses and savings take up, and show how much you have left to spend.")
# user input - what is your monthly income (variable an input)
print(" ")
income = float (input("What is your monthly income: "))
# user input - what is your monthly rent or mortgage (variable an input)
rent_mortgage = float (input("What is your monthly expense for rent or mortgage: "))
# user input - what is your monthly utilities bill (variable an input)
utilities = float (input("What is your monthly expense for utility: "))
# user input - what is your monthly expense on groceries (variable an input)
groceries = float (input("What is your monthly expense for groceries: "))
# user input - what is your monthly expense for transportation (variable an input)
transportation = float(input("What is your monthly expense for transportation: "))
# calculate savings as 10% of income (income * 0.1) (variable not input)
print(" ")
print("Here are your results!")
savings = (income/10)
savings_percentage = (savings/income * 100)
# calculate spending as income-savings-rent-utilities-groceries-transportation (variable not input)
spending = (income-rent_mortgage-utilities-groceries-transportation)
# calculate percent income of rent/mortgage (rent/income * 100) (variable)
rent_mortgage_percentage = (rent_mortgage/income * 100)
# calculate percent income of utilites (utilities/income * 100) (variable)
utilities_percentage = (utilities/income * 100)
# calculate percent income of groceries (groceries/income * 100) (variable)
groceries_percentage = (groceries/income * 100)
# calculate percent income of transportation (transportation/income * 100) (variable)
transportation_percentage = (transportation/income * 100)
# calculate percent income is spending (speneding/income * 100) (variable)
spending_percentage = (spending/income * 100)
# Your rent expenses are $xx.xx, which is xx% of your income. (print)
print(" ")
print("The amount of money you spend on your rent or mortage is $",round(rent_mortgage,2), ", which is ",round(rent_mortgage_percentage, 1),"% of your monthly income.", sep = "")
# Your utilities expenses are $xx.xx, which is xx% of your income. (print)
print("The amount of money you spend on utilities is $",round(utilities,2),  ", which is ",round(utilities_percentage, 1), "% of your monthly income.", sep = "")
# Your groceries expenses are $xx.xx, which is xx% of your income. (print)
print("The amount of money you spend on groceries is $",round(groceries,2), ", which is ",round(groceries_percentage, 1), "% of your monthly income.", sep = "")
# Your transportion expenses are $xx.xx, which is xx% of your income. (print)
print("The amount of money you spend on transportation is $",round(transportation,2), ", which is ",round(transportation_percentage, 1), "% of your monthly income.", sep = "")
# You savings is $xx.xx which is xx% of your income. (print)
print("The amount of money you put into savings is $", round(savings, 2), ", which is ",round(savings_percentage,1), "% of your monthly income.", sep = "")
# Your spending is $xx.xx, which is xx% of your income. (print)
print("The amount of money you have left to spend is $",round(spending,2), ", which is ",round(spending_percentage, 1), "% of your monthly income.", sep = "")
print("Thank you for using Financial Calculator!\nHave a great day!\n")
